julia/Project.toml

```diff
@@ -1,15 +1,8 @@
-name = "TP-IV Exercises"
-authors = ["Matteo Seclì <matteo.secli@epfl.ch>",
-           "Alberto Mercurio <alberto.mercurio@epfl.ch>"]
-
 [deps]
-Arpack = "7d9fca2a-8960-54d3-9f78-7d1dccf2cb97"
 BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
-DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
 IJulia = "7073ff75-c697-5162-941a-fcdaad2a7d2a"
-KrylovKit = "0b1a1467-8014-51b9-945f-bf0ae24f4b77"
-LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
-Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+MKL = "33e6dc65-8f57-5167-99aa-e5a354878fb2"
```

Comment:
Are we sure it wouldn't cause issues for people running locally on anything other than `linux/amd64`? It looks like `osx/amd64` is unsupported as of 2024 (https://github.com/JuliaLinearAlgebra/MKL.jl?tab=readme-ov-file#note-mkl-on-intel-macs), and I expect the package is not available for `*/aarch64` (among which newer Macs) at all. It's not a direct dependency anyway.

Reply:
Yes, we can remove it. It is not cecessary

Reply:
Great!